### PR TITLE
Radarr: fix package update and version bump

### LINKED ISSUE #5363: Radarr installation fails on DS214play & DSM 7.0.X
### Is this a new Bug?

- [X] I checkd that the bug hasn't been reported before

### Package Name

Radarr

### Package Version

v 3.0.0.4204

### Device Model

ds214play

### Device Architecture

i686 (evansport)

### Firmware Version

7.0.1-42218 Update 5

### What happened?

when i try to install the package i get the following error: "Invalid file format. Please contact package developer"

### Reproduction steps

```bash
1. search the package in package center
2. press install button
3. receive the error "Invalid file format. Please contact package developer"
```

### Install Log

```log
radarr.log does not exist in /var/log/packages/
```

### Service Log

_No response_

### Other Logs

_No response_

### PR DESCRIPTION
**Pull request by mreid-tt**
## Description

This includes the following fixes:

1. Upgrade to Radarr 4.3.2.6857
2. Avoid modification of config.xml on package update (see #5514)
3. Update compatible ARMv7 archs
4. Add support for x86 (32-bit) architecture

Fixes #5363

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)
- [x] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [x] Bug fix
- [x] Package update

### CODE REVIEW

File: spk/radarr/src/config.xml [hunk 1]
```diff
@@ -1,8 +1,6 @@
-<?xml version="1.0" encoding="utf-8" standalone="yes"?>
 <Config>
   <Branch>master</Branch>
-  <UpdateMechanism>BuiltIn</UpdateMechanism>
   <LaunchBrowser>False</LaunchBrowser>
-  <Port>8310</Port>
```
Comment (hgy59) [inline spk/radarr/src/config.xml:6]: I like to have the port definition here (despite it is the default).
This is good for documentation (and prevents troubles, when, for what reason ever, the default should change).
Only the Port should be kept, not the SslPort (except the package would support the ssl port too).

Comment (mreid-tt) [reply spk/radarr/src/config.xml:6]: hmm, okay i'll add it in but we need to be consistent with the *arr apps. the release we worked on for sonarr doesn't have the port number

**Comment by mreid-tt** [comment]
@hgy59 thanks for your review, let me know if there is anything further needed on this PR before merging.

**Comment by bakerboy448** [comment]
New radarr master coming shortly with the update_required fix https://github.com/Radarr/Radarr/tree/master

**Comment by mreid-tt** [comment]
> New radarr master coming shortly with the update_required fix https://github.com/Radarr/Radarr/tree/master

Thanks for the heads up. I'll update the 'digests' as soon as it is released.

**Comment by mreid-tt** [comment]
@bakerboy448, the new version 4.3.2.6857 of radarr is now included.
@hgy59, anything else needed before this is published?

**Comment by bakerboy448** [comment]
Looks like this may have an unintentional side affect with trying to avoid radarr app downgrades....it also doesn't do upgrades...if the user is on a legacy unsupported radarr version

somehow it appears a user is running radarr v3.2.2 which is no longer supported, yet supposedly running the new v4 package.

https://www.reddit.com/r/radarr/comments/11ek129/issue_with_newest_update_libmediainfoso0/

**Comment by mreid-tt** [comment]
> somehow it appears a user is running radarr v3.2.2 which is no longer supported, yet supposedly running the new v4 package.

I would recommend redirecting the user to the SynoCommunity discord or opening an issue on GitHub to diagnose further.

**Pull request merged**